Source/RESTyard.Client/Exceptions/ProblemDetails.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace RESTyard.Client.Exceptions
+{
+    /// <summary>
+    /// Copy-Paste of Microsoft.AspNetCore.Mvc to not force including Microsoft.AspNetCore.Mvc into the client
+    /// A machine-readable format for specifying errors in HTTP API responses based on https://tools.ietf.org/html/rfc7807.
```

Comment:
I decided since this file is a copy-paste, to not change the description.
But in theory the client can do other protocols than HTTP. Should this description still mention it? The RFC is specific to HTTP, so maybe leaving it is fine here?